common/src/main/java/io/druid/guice/DruidSecondaryModule.java

```diff
@@ -75,24 +76,36 @@ public void configure(Binder binder)
     binder.bind(JsonConfigurator.class).toInstance(jsonConfigurator);
   }
 
-  @Provides @LazySingleton @Json
-  public ObjectMapper getJsonMapper(final Injector injector)
+  @Provides
+  @LazySingleton
+  public InjectableValues getGuiceInjectableValues(final Injector injector)
```

Comment:
Ah ok, well if we are making this change as part of #1092 already, then let's remove those changes completely and let #1092 deal with it.